src/network/networkd-network.c

```diff
@@ -165,7 +165,7 @@ static int network_load_one(Manager *manager, const char *filename) {
                 return r;
 
         LIST_FOREACH(routes, route, network->static_routes) {
-                if (!route->family) {
+                if (!IN_SET(route->family, AF_INET, AF_INET6, AF_UNSPEC)) {
```

Comment:
Hm... doesn't this change the logic to not care at all if Gateway= is set? When would this error be triggered?